src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -98,4 +98,128 @@ describe('recommendations list table', () => {
       .find('tbody[role=rowgroup]')
       .should('have.length', 9);
   });
+
+  it('Recommendations table should have 4 sortable columns', () => {
+    cy.get('table[class="pf-c-table pf-m-grid-md pf-m-compact"]').should(
+      'have.length',
+      1
+    );
+    cy.get('th[class="pf-c-table__sort pf-m-width-70"]').should(
+      'have.length',
+      1
+    );
+    cy.get('th[class="pf-c-table__sort pf-m-width-10"]').should(
+      'have.length',
+      3
+    );
+  });
+
+  it('Recommendation table sort the data by Name first sort', () => {
+    cy.get(RECS_LIST_TABLE)
+      .get('span[class=pf-c-table__sort-indicator]')
+      .first()
+      .click({ force: true });
+    cy.get('td[data-label=Name]')
+      .find('a')
+      .should(($a) => {
+        expect($a[0]).to.have.text(
+          ' Additional risks would occur possibly when having the masters defined as machinesets '
+        );
+      });
+  });
+
+  it('Recommendation table sort the data by Name second sort', () => {
+    cy.get(RECS_LIST_TABLE)
+      .get('span[class=pf-c-table__sort-indicator]')
+      .first()
+      .click({ force: true })
+      .click({ force: true });
+    cy.get('td[data-label=Name]')
+      .find('a')
+      .should(($a) => {
```

Comment:
The same story: please see my comment above and update for the rest of the tests if applicable.